gcloud/bigtable/row.py

```diff
@@ -42,6 +42,43 @@ def __init__(self, row_key, table, filter_=None):
         self._row_key = _to_bytes(row_key)
         self._table = table
         self._filter = filter_
+        self._rule_pb_list = []
```

Comment:
There will likely be a merge conflict but it's building towards the same object.